moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -164,7 +170,7 @@ ompl::base::PlannerPtr ompl_interface::MultiQueryPlannerAllocator::allocatePlann
   // Try to initialize planner with loaded planner data
   if (load_planner_data)
   {
-    ROS_INFO("Loading planner data");
+    ROS_INFO_NAMED(LOGNAME, "Loading planner data");
```

Comment:
Resolved because of my comment higher up in the code:

>This is not my code, but I feel an info message is justified. You have to ask MoveIt explicitly to reuse a planning graph, so it seems nice to have some feedback that it actually happened, no? See #1799 for the related PR.